graph.cpp

```diff
@@ -15,6 +18,13 @@ Graph::Graph(uint64_t num_nodes): num_nodes(num_nodes) {
     supernodes[i] = new Supernode(num_nodes,seed);
     parent[i] = i;
   }
+  num_updates = 0; // REMOVE this later
```

Comment:
do we need this?

Reply:
Maybe, maybe not. It's been helpful to me to have it but it's definitely not strictly necessary

Reply:
What would be its purpose in production?

Reply:
Might not have one. Depends if we think it's useful now. Although perhaps it would be nice to be able to verify that the total number of updates to the graph is what we expected it would be.

Reply:
Sure, I guess we can keep it for now.